api/package.json

```diff
@@ -1,11 +1,12 @@
 {
   "scripts": {
     "prestart": "yarn prisma db pull && yarn prisma generate",
+    "generate": "bun prisma generate",
     "start": "yarn ts-node-dev --debug --files --exit-child src/server.ts",
     "prod": "yarn ts-node --files src/server.ts",
```

Comment:
Should these also use `bun`? Also why is `generate` now separate?

Reply:
For some reason it's necessary with bun's install - I think it's cuz it doesn't yarn's postinstall prisma client build step.